packages/init/__tests__/configure-project.test.ts

```diff
@@ -260,7 +260,7 @@ describe("configure-project", () => {
 
     expect(successFn).not.toHaveBeenCalled();
     expect(failFn).toHaveBeenCalled();
-    expect(MockTracker).not.toHaveBeenCalled();
+    expect(MockTracker).toHaveBeenCalled();
```

Comment:
can you also assert the payload it sent? 
See how it's done in the file below:
```expect(MockTracker).toHaveBeenCalledTimes(1);
    expect(MockTracker.mock.calls[0][0]).toEqual({
      anonymousId: "uuid",
      event: "SliceMachine Init End",
      properties: {
        framework: Models.Frameworks.next,
        repo: "repoName",
        result: "success",
      },
      context: { groupId: { Repository: "repoName" } },
    });```

Reply:
or at least test the event and properties: 
```expect(MockTracker.mock.calls[0][0].event).toEqual(...))```
```expect(MockTracker.mock.calls[0][0]. properties).toEqual(...))```

Reply:
This is already tested in `tracker.test.ts`. Do you think it's still necessary?

Reply:
updated